# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/hotfix.patch
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-55.0/media/libstagefright/binding/mp4parse_capi/Cargo.toml.vanilla	2017-08-19 23:06:45.845758246 +0200
+++ firefox-55.0/media/libstagefright/binding/mp4parse_capi/Cargo.toml	2017-08-19 23:06:49.401758221 +0200
@@ -18,7 +18,7 @@
   "*.mp4",
 ]
 
-build = false
+#build = false
 
 [dependencies]
 byteorder = "1.0.0"
